src/app/fdctl/run/tiles/fd_gossip.c

```diff
@@ -356,6 +356,15 @@ gossip_signer( void *        signer_ctx,
   fd_keyguard_client_sign( ctx->keyguard_client, signature, buffer, len, sign_type );
 }
 
+static void
+during_housekeeping( void * _ctx ) {
```

Comment:
This can just be `fd_gossip_tile_ctx_t * ctx`

Reply:
oh nice didn't catch that